test/index.test.js

```diff
@@ -1,10 +1,44 @@
-// You can import your modules
-// const index = require('../index')
+const { Application } = require('probot')
+// Requiring our app implementation
+const plugin = require('..')
+
+const issuesOpenedPayload = require('./fixtures/issues.opened.json')
+
 
 test('that we can run tests', () => {
   // your real tests go here
   expect(1 + 2 + 3).toBe(6)
 })
 
+describe('my probot app', () => {
+  let app
```

Comment:
Can this be `let app, github`? Or do you prefer it like this (I prefer the first, but I won't die on this hill)

Reply:
done! We should also update this in the docs and anywhere else it's used for consistency